package.json

```diff
@@ -14,7 +14,6 @@
   "source": "./app/scripts/index.ts",
   "main": "lib/main.js",
   "module": "lib/module.js",
-  "types": "lib/index.d.ts",
```

Comment:
I had to get rid of this.. otherwise the build for the library would keep failing with some `@parcel-transformer-typescript-types` error on very random types. This was a headache. This is probably so parcel can compile the types to export by extracting the library's types and generating a `.d.ts` file but I think we can do without it. Any concerns or unforeseen risks if we do this?

Reply:
I might be wrong, but the first concern that comes to my mind is not having any TS declarations. So any project that would use our veda-ui will not have type information and will treat all imports from our package as having an `any` type.

I've also seen some TS errors on the main branch when running the `buildlib`, but the build succeeds there at the end. Maybe we could run the lib build command between the `main` branch and this one to narrow down which error fails the build?